modules/reference/pages/public-metrics-reference.adoc

```diff
@@ -497,6 +497,32 @@ Latency of produce/consume requests per broker. This duration measures from when
 
 ---
 
+=== redpanda_kafka_quotas_client_quota_throttle_time
+
+Client quota throttling delay, in seconds, per rule and quota type.
+
+*Type*: histogram
+
+*Labels*:
+
+* quota_rule=("not_applicable" | "kafka_client_default" | "cluster_client_default" | "kafka_client_prefix" | "cluster_client_prefix" | "kafka_client_id")
+* quota_type=("produce_quota" | "fetch_quota" | "partition_mutation_quota")
+
```

Comment:
@kbatuigas please check that these render consistent with the other metrics on this page. See [here](https://deploy-preview-592--redpanda-docs-preview.netlify.app/24.2/reference/public-metrics-reference/#redpanda_kafka_quotas_client_quota_throttle_time) compared to the one above it.